WordPress/Classes/ViewRelated/Blog/SiteSettingsViewController.m

```diff
@@ -811,41 +810,49 @@ - (void)showDeleteSiteForBlog:(Blog *)blog
 - (void)tableView:(UITableView *)tableView didSelectInAdvancedSectionRow:(NSInteger)row
 {
     switch (row) {
-        case SiteSettingsAdvancedStartOver: {
+        case SiteSettingsAdvancedStartOver:
             [self showStartOverForBlog:self.blog];
-        } break;
-        case SiteSettingsAdvancedDeleteSite: {
+            break;
+
+        case SiteSettingsAdvancedDeleteSite:
             [self showDeleteSiteForBlog:self.blog];
-        } break;
+            break;
     }
 }
 
 - (void)tableView:(UITableView *)tableView didSelectRowAtIndexPath:(NSIndexPath *)indexPath
 {
+    [tableView deselectSelectedRowWithAnimation:YES];
```

Comment:
This was a deliberate change as its absence looked odd, and its addition matches the behavior in most other table views throughout the app.  
> but general behavior to have a cell stay selected until returning from a detail view.

I wouldn't say this is the general behavior in the app today (based on a quick `ack`), but maybe it should be.  I hadn't noticed the cell highlight dismissing after tapping the back button but now I can't _unsee_ it. :P Let's revert this guy.